tests/Feature/BookTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Book;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Testing\Fluent\AssertableJson;
+use Tests\TestCase;
+
+class BookTest extends TestCase
+{
+
+    use DatabaseMigrations;
+    /**
+     * A basic feature test example.
+     */
+    public function test_returnBook_success(): void
+    {
+        Book::factory()->create();
```

Comment:
You can do 
```
$book = Book::factory()->create();
$book->whatever = 'huh?';
$book->save();
```

To control the data in your test db. It's probably better to do this in the test vs in the factory. If someone made changes to the factory, all the tests that depend on specific data from the factory will start failing.

Reply:
You can also do it with the same syntax you used in the last test here